src/sagas/delete-project.saga.test.js

```diff
@@ -0,0 +1,170 @@
+import electron from 'electron';
+import { call, put, cancel, select, takeEvery } from 'redux-saga/effects';
+
+import rootSaga, {
+  deleteProject,
+  getNextProjectId,
+} from './delete-project.saga';
+
+import {
+  SHOW_DELETE_PROJECT_PROMPT,
+  finishDeletingProjectFromDisk,
+  selectProject,
+  createNewProjectStart,
+} from '../actions';
+import { getProjectsArray } from '../reducers/projects.reducer';
+
+jest.mock('electron', () => ({
```

Comment:
ah good idea!

I think I missed the creation of `__mocks__`. Our pattern so far is around colocated dot notation (so, `thing.tests.js` instead of `__tests__/thing.js`). What do you think about doing something similar? So we'd just have `src/electron.mocks.js`

I don't have strong feelings either way but I like the idea of being consistent.

Reply:
I could be wrong but I believe the `__mocks__` directory is a Jest-ism, it's automatically detected when `jest.mock()` is called on a module in the parent directory.

Reply:
Ohh ok! that's fine then.